go-controller/pkg/ovn/policy_test.go

```diff
@@ -462,8 +462,8 @@ var _ = Describe("OVN NetworkPolicy Operations", func() {
 					},
 				)
 				nPodTest.populateLogicalSwitchCache(fakeOvn)
-				fakeOvn.controller.WatchPods()
 				fakeOvn.controller.WatchNamespaces()
+				fakeOvn.controller.WatchPods()
```

Comment:
we've gone back and forth several times moving these in policy_test and I don't think its a good idea. It always seems to catch bugs with our code and now pods should always wait for the namespace.

Reply:
@trozet not sure what your suggestion here is. Whatever the ordering is in pkg/ovn/ovn.go (which is current Namespaces then Pods) is what the testcases must also use.

Reply:
my suggestion is not to change this, because keeping it help to expose bugs when we get events out of order. We dont need to start namespaces before pods, because pods should wait until namespace is created.

Reply:
@trozet they do, but they block forever waiting for a namespace to show up, and fail the test. This reduces test flakes becuase it's not a condition we'll ever encounter, because our ovn.go code calls them this way. I don't get why having them not in the same order would help at all?

Reply:
Added testcase to ensure that pod is retried if namespace doesn't exist when the first pod Add happens.